WooCommerce/Classes/ViewRelated/Orders/Order Creation/Taxes/NewTaxRateSelectorView.swift

```diff
@@ -37,65 +37,46 @@ struct NewTaxRateSelectorView: View {
                     .footnoteStyle()
                     .multilineTextAlignment(.leading)
                     .padding([.leading, .trailing], Layout.generalPadding)
-                    .padding(.bottom, Layout.taxRatesSectionTitleBottomPadding)
+                    .padding([.top, .bottom], Layout.taxRatesSectionTitleVerticalPadding)
 
                 Divider()
 
-                ForEach(viewModel.demoTaxRates, id: \.title) { taxRate in
-                    HStack {
-                        Button(action: { }) {
-                            AdaptiveStack(horizontalAlignment: .leading, spacing: Layout.generalPadding) {
-                                Text(taxRate.title)
-                                    .foregroundColor(Color(.text))
-                                    .multilineTextAlignment(.leading)
-                                    .frame(maxWidth: .infinity, alignment: .leading)
-
-                                Text(taxRate.value)
-                                    .foregroundColor(Color(.text))
-                                    .multilineTextAlignment(.trailing)
-                                    .frame(width: nil, alignment: .trailing)
+                switch viewModel.syncState {
+                    case .results:
+                        RefreshableInfiniteScrollList(isLoading: viewModel.shouldShowBottomActivityIndicator,
```

Comment:
I can see this is pre-existing, and probably the cause of the various paging/refreshing issues I mentioned in my comment. Perhaps a separate PR to improve it is warranted? It's not been touched for a year and a half, so given the pace of SwiftUI development, it might be possible to do better now....

Reply:
I removed that component in 210b19437e649b0372b11146bce27d3094c0fb2c As you mentioned, it was causing several issues due to the refresh mechanism, which is not necessary for this task. Fixing those issues is not worth the effort compared to the benefits it might bring.